CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.29.0
+### Added
+* Service: PII is now removed from Jobs, Tasks, and Repros after 18 months.  [#1051](https://github.com/microsoft/onefuzz/pull/1051)
+* Service: Unused notifications are now removed after 18 months.  [#1051](https://github.com/microsoft/onefuzz/pull/1051)
+
+### Changed
+* Service: SignalR events are routed through an Azure Storage Queue to prevent SignalR outages from impacting the entire service.  [#1100](https://github.com/microsoft/onefuzz/pull/1100), [#1102](https://github.com/microsoft/onefuzz/pull/1102)
+* Service: Pools and Scalesets set to `shutdown` can now be set to `halt`.  [#1104](https://github.com/microsoft/onefuzz/pull/1104)
```

Comment:
Would it be better to put this under "Fixed"?

Reply:
```suggestion
```